### PR TITLE
[ja] Translate concepts/scheduling-eviction/pod-priority-preemption/ into Japanese

### PR DESCRIPTION
**Pull request by Gekko0114**
Translated [concepts/scheduling-eviction/pod-priority-preemption/](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md) into Japanese.

I found that previously concepts/configuration/pod-priority-preemption was moved to concepts/scheduling-eviction/pod-priority-preemption.
So first I moved [ja/docs/concepts/configuration/pod-priority-preemption](https://github.com/kubernetes/website/blob/main/content/ja/docs/concepts/configuration/pod-priority-preemption.md) to ja/docs/concepts/scheduling-eviction/pod-priority-preemption. Then I updated some points.

This is my first time contributing to this repository. So if there are any mistakes, please let me know.
Thanks,

Close https://github.com/kubernetes/website/issues/39104

### CODE REVIEW

**Comment by Gekko0114** [comment]
Hi @kakts @t-inu,

Could you review my PR?
This is my first PR for this repo. If I did a wrong thing, please let me know.
Thanks in advance,

**Review by t-inu — COMMENTED** [review]
Welcome and thanks for your contribution.
I left 2 comments, please check.

**Comment by Gekko0114** [comment]
Thank you so much for your detailed review. I will revise it and push again.

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 1]
```diff
@@ -227,14 +226,22 @@ Podがその期間内に終了しない場合、強制終了されます。プ
 Podの優先度と{{< glossary_tooltip text="QoSクラス" term_id="qos-class" >}}は直交する機能で、わずかに相互作用がありますが、デフォルトではQoSクラスによる優先度の設定の制約はありません。スケジューラーのプリエンプションのロジックはプリエンプションの対象を決めるときにQoSクラスは考慮しません。
 プリエンプションはPodの優先度を考慮し、優先度が最も低いものを候補とします。より優先度の高いPodは優先度の低いPodを追い出すだけではプリエンプトを起こしたPodのスケジューリングに不十分な場合と、`PodDisruptionBudget`により優先度の低いPodが保護されている場合のみ対象になります。
 
-QoSとPodの優先度の両方を考慮するコンポーネントは[リソース不足によりkubeletがPodを追い出す](/docs/tasks/administer-cluster/out-of-resource/)のみです。
-kubeletは追い出すPodの順位付けを次の順で行います。枯渇したリソースを要求以上に使用しているか、優先度、枯渇したリソースの消費量の複数のPodの要求に対する相対値。
-詳細は[エンドユーザーのPodの追い出し](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods)を参照してください。
+kubeletは[node-pressureによる退避](/docs/concepts/scheduling-eviction/node-pressure-eviction/)を行うPodの順番を決めるために、優先度を利用します。
+kubeletは追い出すPodの順位付けを次の順で行います。
```
[the remark this answers is not in the document]
Comment (Gekko0114) [reply content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md:230]: コメントありがとうございます。修正しました。

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 2]
```diff
@@ -143,14 +143,13 @@ Podが作成されると、スケジューリング待ちのキューに入り
 
 Pod PがノードNのPodをプリエンプトした場合、ノードNの名称がPのステータスの`nominatedNodeName`フィールドに設定されます。このフィールドはスケジューラーがPod Pのために予約しているリソースの追跡を助け、ユーザーにクラスターにおけるプリエンプトに関する情報を与えます。
 
-Pod Pは必ずしも「指名したノード」へスケジューリングされないことに注意してください。Podがプリエンプトされると、そのPodは終了までの猶予期間を得ます。スケジューラーがPodの終了を待つ間に他のノードが利用可能になると、スケジューラーは他のノードをPod Pのスケジューリング先にします。この結果、Podの`nominatedNodeName`と`nodeName`は必ずしも一致しません。また、スケジューラーがノードNのPodをプリエンプトさせた後に、Pod Pよりも優先度の高いPodが来た場合、スケジューラーはノードNをその新しい優先度の高いPodへ与えます。このような場合は、スケジューラーはPod Pの`nominatedNodeName`を消去します。これによって、スケジューラーはPod Pが他のノードのPodをプリエンプトさせられるようにします。
+Pod Pは必ずしも「指名したノード」へスケジューリングされないことに注意してください。Podがプリエンプトされると、そのPodは終了までの猶予期間を得ます。スケジューラーがPodの終了を待つ間に他のノードが利用可能になると、スケジューラーは他のノードをPod Pのスケジューリング先にする可能性があります。この結果、Podの`nominatedNodeName`と`nodeName`は必ずしも一致しません。また、スケジューラーがノードNのPodをプリエンプトさせた後に、Pod Pよりも優先度の高いPodが来た場合、スケジューラーはノードNをその新しい優先度の高いPodへ与えます。このような場合は、スケジューラーはPod Pの`nominatedNodeName`を消去します。これによって、スケジューラーはPod Pが他のノードのPodをプリエンプトさせられるようにします。
```
[the remark this answers is not in the document]
Comment (Gekko0114) [reply content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: コメントありがとうございます。修正しました。

**Comment by Gekko0114** [comment]
Hi @kakts @t-inu,

Thank you so much for your comments. I fixed it.
Please review my PR again.

Thanks,

**Review by t-inu — APPROVED** [review]
/lgtm

**Comment by t-inu** [comment]
/assign @nasa9084
Could you check this PR?

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 3]
```diff
@@ -227,14 +226,22 @@ Podがその期間内に終了しない場合、強制終了されます。プ
 Podの優先度と{{< glossary_tooltip text="QoSクラス" term_id="qos-class" >}}は直交する機能で、わずかに相互作用がありますが、デフォルトではQoSクラスによる優先度の設定の制約はありません。スケジューラーのプリエンプションのロジックはプリエンプションの対象を決めるときにQoSクラスは考慮しません。
 プリエンプションはPodの優先度を考慮し、優先度が最も低いものを候補とします。より優先度の高いPodは優先度の低いPodを追い出すだけではプリエンプトを起こしたPodのスケジューリングに不十分な場合と、`PodDisruptionBudget`により優先度の低いPodが保護されている場合のみ対象になります。
 
-QoSとPodの優先度の両方を考慮するコンポーネントは[リソース不足によりkubeletがPodを追い出す](/docs/tasks/administer-cluster/out-of-resource/)のみです。
-kubeletは追い出すPodの順位付けを次の順で行います。枯渇したリソースを要求以上に使用しているか、優先度、枯渇したリソースの消費量の複数のPodの要求に対する相対値。
-詳細は[エンドユーザーのPodの追い出し](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods)を参照してください。
+kubeletは[node-pressureによる退避](/docs/concepts/scheduling-eviction/node-pressure-eviction/)を行うPodの順番を決めるために、優先度を利用します。QoSクラスを使用して、最も退避される可能性の高いPodの順番を推定することができます。
+kubeletは追い出すPodの順位付けを次の順で行います。
+
+
```
Comment (nasa9084) [inline content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md:232]:
```suggestion
```

**Review by nasa9084 — COMMENTED** [review]
Thank you for your first contribution :tada:
I left small suggest :bow:

**Comment by Gekko0114** [comment]
Hi @kakts @t-inu @nasa9084,

Thanks for your comments. I fixed your review comments. So could you approve this PR?
Thanks,

**Review by t-inu — APPROVED** [review]
/lgtm

**Comment by nasa9084** [comment]
/approve
Thank you for your contribution!!

**Pull request merged**